metron-deployment/inventory/full-dev-platform/group_vars/all

```diff
@@ -15,21 +15,38 @@
 #  limitations under the License.
 #
 
-#Ambari variables
+# which services should be started?
+services_to_start:
+  - mysql
+  - elasticsearch
+  - pcap-service
+  - metron-ui
+  - yaf
+  - snort
+  - snort-logs
+  - bro
+  - pcap-replay
+  - yaf-parser
+  - bro-parser
+  - snort-parser
+  - enrichment
```

Comment:
Since both "Quick Dev Platform" and "Full Dev Platform" use the same properties at `metron-deployment/inventory/full-dev-platform`, we cannot have each start different services.  This would be possible, if we separate out the properties that each use.

As a side note, I think we need to revisit what lives in `metron-deployment/inventory`.